### ISSUE TITLE
Inconsistent result

### ISSUE THREAD

**Issue by fransonf** [junyanz/pytorch-CycleGAN-and-pix2pix#979]
Hello! I've been doing experiments with CycleGAN regarding the number of images needed to produce decent result for my translation task. 

The problem with this is that two identical trainings (same hyperparameters, architecture, image size, dataset, etc...) yield very different result, i.e. it is difficult reproducing the same result over and over again. As an example, I got very good results using 2500 images one run but poor results the second time I tried with 2500 images. How can this best be explained?

Thanks a lot in advance.

**Comment by junyanz** [comment]
Both model initialization and training are stochastic. Results might vary across different runs and checkpoints. If you have an evaluation metric for your tasks, you can compute the average metrics over 5 or 10 runs. You can also use the metric to choose the best model.

**Comment by fransonf** [comment]
Thanks!
